Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -807,17 +815,18 @@ class UnrealResourcePreparer
       }
     }
 
+    CesiumGltf::SharedAsset<CesiumGltf::ImageCesium> imageAsset(image);
```

Comment:
Won't this copy the Image? That seems expensive. As I mentioned when I reviewed the cesium-native side of this, I don't really see much need for this whole SharedAsset / SharedContainer design. Having `Image` hold `IntrusivePointer<ImageCesium>` seems like it should be sufficient.